pkg/cli/verifier/envoy_config.go

```diff
@@ -248,3 +257,120 @@ func getFilterForProtocol(protocol string) string {
 		return ""
 	}
 }
+
+func (v *EnvoyConfigVerifier) verifyDestination() Result {
+	result := Result{
+		Context: fmt.Sprintf("Verify Envoy config on destination for traffic [%s]", v.configAttr),
```

Comment:
In `verifySource()`, the config attribute is prefixed with a colon

`Context: fmt.Sprintf("Verify Envoy config on source for traffic: %s", v.configAttr)`

In `verifyDestination()` we are using brackets around the config attributes. Is this difference intentional?

Reply:
Good catch, fixed!